tests/integration/define.py

```diff
@@ -0,0 +1,324 @@
+from conditions import (
```

Comment:
I think we should change the name of this file.
'define' is too generic and could clash with other libraries (maybe? down the road). maybe test_definitions.py